app/Http/Requests/BuildingRequest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class BuildingRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
```

Comment:
dit is helaas nog over van de kwestie "De combinatie van de Afkorting+gebouwnaam moet uniek zijn". Niet zo netjes van mij om dit niet te ontcommenten.